cfn-resources/access-list-api-key/test/cfn-test-create-inputs.sh

```diff
@@ -25,10 +25,16 @@ if [ ${MONGODB_ATLAS_PROFILE+x} ];then
 echo "profile set to ${MONGODB_ATLAS_PROFILE}"
 Profile=${MONGODB_ATLAS_PROFILE}
 fi
-# Check MONGODB_ATLAS_ORG_ID is set
-if [ -z "${MONGODB_ATLAS_ORG_ID+x}" ];then
-  echo "MONGODB_ATLAS_ORG_ID must be set"
-  exit 1
+
+# Check if MONGODB_ATLAS_ORG_ID is set
+if [ -z "${MONGODB_ATLAS_ORG_ID+x}" ]; then
+  # Check if ATLAS_ORG_ID is set as a fallback
+  if [ -z "${ATLAS_ORG_ID+x}" ]; then
+    echo "MONGODB_ATLAS_ORG_ID or ATLAS_ORG_ID must be set"
```

Comment:
this is something that i have asked on slack here https://mongodb.slack.com/archives/C044FEU3F9C/p1695757906182339 i did not want to remove the other, it seems that in one contract testing is using MONGODB_ATLAS_ORG_ID but in the publishing flow is using  ATLAS_ORG_ID, i just asked what sould we do, and for now i just validated both, in case one does not exists im using the other, and in cas non exists, i just thow na error

Reply:
From Colm feedback, I would simplify and rely on one variable. In this case we can use ATLAS_ORG_ID which I understand is the one that is working correctly.

Reply:
We use `MONGODB_ATLAS_ORG_ID` across the integrations (TF, acceptance tests, and the Atlas CLI). To maintain this consistency, I would use `MONGODB_ATLAS_ORG_ID` exclusively.

Reply:
if that is the case andrea, we will have to modify the publishing flow, is kind of a big change, i would do that in a different pull request

Reply:
it makes sense. Feel free to create a ticket to track this work. Thanks!